src/coreclr/jit/emitfmtswasm.h

```diff
@@ -28,6 +28,8 @@ enum ID_OPS
 
 IF_DEF(NONE,    IS_NONE, NONE)
 IF_DEF(OPCODE,  IS_NONE, NONE) // <opcode>
+IF_DEF(TYPE,    IS_NONE, NONE) // <opcode> <type 0x40>
+IF_DEF(LABEL,   IS_NONE, NONE) // <ULEB128 immediate>
```

Comment:
What instruction is this supposed to represent? I see there is a pseudo-instruction `INS_label`, but it's unused and it's not clear what would it be used for.